conduct/code_of_conduct.md

```diff
@@ -0,0 +1,134 @@
+# Project Jupyter Code of Conduct and Community Guidelines
```

Comment:
Shorten name to "Project Jupyter Code of Conduct" to be explicit and match naming of document to Django's CoC on which this is based.